patches/chrome-BUILD.gn.patch

```diff
@@ -66,33 +66,24 @@ index b8c378b044d9142c9936c97e686fb6bfcb37c918..58f9f4c565cd52c6f7c4b334da4cd91e
            "-v",
            rebase_path(chrome_version_file, root_build_dir),
            "-g",
-@@ -781,7 +794,7 @@ if (is_win) {
-         # framework itself, that would cause a cyclical dependency. Instead,
-         # this dependency directly copies the file into the framework's
-         # resources directory.
--        public_deps += [ ":chrome_framework_widevine_signature" ]
-+        #public_deps += [ ":chrome_framework_widevine_signature" ]
-       } else {
-         sources += [ "$root_out_dir/Widevine Resources.bundle" ]
-         public_deps += [ ":widevine_resources_bundle" ]
-@@ -1184,7 +1197,7 @@ if (is_win) {
-         bundle_resources_dir = "$bundle_contents_dir/Resources"
+@@ -823,7 +836,7 @@ if (is_win) {
+       # framework itself, that would cause a cyclical dependency. Instead,
+       # this dependency directly copies the file into the framework's
+       # resources directory.
+-      public_deps += [ ":chrome_framework_widevine_signature" ]
++      #public_deps += [ ":chrome_framework_widevine_signature" ]
```

Comment:
can we change this to `public_deps -= [ ":chrome_framework_widevine_signature" ]` so it's an addition instead of a modification?

Reply:
actually don't worry about it, we'll clean this stuff up separately